src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.0.0.2821")]
+[assembly: AssemblyVersion("2.0.0.2906")]
```

Comment:
would you mind removing this file from your PR?

Reply:
Yeah, Would you revert this file? Other changes looks fine.

Reply:
ok! Sorry for my distraction

Reply:
How can I revert this file without do another commit?